SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -117,8 +117,7 @@ protected T init(final Activity activity) throws IllegalArgumentException {
             saveUniqueUserId(resolvedContext, userId);
             SwrveLogger.i(LOG_TAG, "Your user id is: " + userId);
 
-            // Generate default urls for the given app id
-            config.generateUrls(appId);
+            config.setUseHttpsForEventsUrl(true);
```

Comment:
Doesn't this depend on the boolean value at the end.

IE setUseHttpsForEventsUrl(false) will set it to use http.

Does that answer your question?